pkg/controller/directvolumemigration/rsync.go

```diff
@@ -195,16 +256,71 @@ func (t *Task) getRsyncTransferServerMutations(client compat.Client, namespace s
 		return nil, liberr.Wrap(err)
 	}
 	containerMutation.Resources = resourceRequirements
-	trueBool := bool(true)
-	runAsUser := int64(0)
-	securityContext := &corev1.SecurityContext{
-		Privileged:             &isPrivileged,
-		ReadOnlyRootFilesystem: &trueBool,
-		RunAsUser:              &runAsUser,
-		Capabilities: &corev1.Capabilities{
-			Drop: []corev1.Capability{"MKNOD", "SETPCAP"},
-		},
+	migration, err := t.Owner.GetMigrationForDVM(client)
+	if err != nil {
+		return nil, liberr.Wrap(err)
 	}
+	securityContext := &corev1.SecurityContext{}
```

Comment:
Lot of the logic seems to be repeated for the server containers, can we write a function instead?